docs/faq.md

```diff
@@ -33,7 +33,16 @@ library(SingleCellExperiment)
 scpca_sample <- readRDS("SCPCL000000_filtered.rds")
 ```
 
-## What is the difference between a sample and a library?
+## What is the difference between participants, samples, and libraries?
+
+The `participant_id` indicates the participant, or patient, from which a sample, or collection of samples, is obtained from. 
+A sample ID, labeled as `scpca_sample_id` and indicated by the prefix `SCPCS`, represents a unique tissue that was collected from a participant. 
+For example, one participant may have a sample collected both at initial diagnosis and at relapse.
+This would result in two unique sample ID's with the same participant ID. 
+
+The library ID, labeled as `scpca_library_id` and indicated by the prefix `SCPCL`, represents a single emulsion and droplet generation using the 10X Genomics workflow from a given sample. 
+In most cases, each sample will only have one corresponding library.
```

Comment:
Realizing that as we get bulk, this may no longer be true.
```suggestion
In most cases, each sample will only have one corresponding single-cell library, and may also have an associated bulk RNA-seq library.
```